go.mod

```diff
@@ -41,20 +41,20 @@ require (
 	go.opentelemetry.io/otel/metric v1.19.0
 	go.opentelemetry.io/otel/sdk v1.19.0
 	go.opentelemetry.io/otel/sdk/metric v1.19.0
-	go.temporal.io/api v1.26.2-0.20231129165614-630d88440548
-	go.temporal.io/sdk v1.25.2-0.20231129171107-288a04f72145
+	go.temporal.io/api v1.26.1-0.20240103185939-608bdd111e4b
```

Comment:
It looks like we downgraded?

Reply:
It did not; we retracted the v1.26.1 version. It's... weird. I don't think Go handles it gracefully

Reply:
Oh, I see!